plugin/runtime/communication.py

```
@@ -0,0 +1,314 @@
+    # 异步相关资源
+    _pending_futures: Dict[str, asyncio.Future] = field(default_factory=dict)
+    _result_consumer_task: Optional[asyncio.Task] = None
+    _message_consumer_task: Optional[asyncio.Task] = None
+    _shutdown_event: Optional[asyncio.Event] = None
+    _executor: Optional[ThreadPoolExecutor] = None
+    _message_target_queue: Optional[asyncio.Queue] = None  # 主进程的消息队列
+    
+    def __post_init__(self):
+        """初始化异步资源"""
+        self._shutdown_event = asyncio.Event()
+        # 为每个插件创建独立的线程池，避免阻塞
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)